absearch/server.py

```diff
@@ -142,9 +169,11 @@ def add_user_to_cohort(**kw):
             raise HTTPError(status=404)
 
         cohort = res.get('cohort', 'default')
-        cohort = '.'.join(['cohorts', kw['locale'],
-                           kw['territory'], cohort])
-        app._statsd.incr(cohort)
+        if cohort != 'default':
```

Comment:
the stats we emit are not used right now, depending on what you mean by 'advertise' - I think we're ok if we break old stats